feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go

```diff
@@ -228,66 +228,70 @@ func TestBasicStaticRouteSupport(t *testing.T) {
 
 func TestStaticRouteAddRemove(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
-	configureDUT(t, dut)
-
-	ate := ondatra.ATE(t, "ate")
-	top := gosnappi.NewConfig()
-	configureOTG(t, ate, top)
-
-	ate.OTG().PushConfig(t, top)
-	ate.OTG().StartProtocols(t)
-	defer ate.OTG().StopProtocols(t)
-	otgutils.WaitForARP(t, ate.OTG(), top, "IPv4")
-
-	prefix := ipAddr{address: v4Route, prefix: v4RoutePrefix}
-	b := &gnmi.SetBatch{}
-	sV4 := &cfgplugins.StaticRouteCfg{
-		NetworkInstance: deviations.DefaultNetworkInstance(dut),
-		Prefix:          prefix.cidr(t),
-		NextHops: map[string]oc.NetworkInstance_Protocol_Static_NextHop_NextHop_Union{
-			"0": oc.UnionString(atePort2.IPv4),
-			"1": oc.UnionString(atePort3.IPv4),
-		},
-	}
-	if _, err := cfgplugins.NewStaticRouteCfg(b, sV4, dut); err != nil {
-		t.Fatalf("Failed to configure IPv4 static route: %v", err)
-	}
-	b.Set(t, dut)
-	validateStaticRoute(t, dut, prefix.cidr(t), sV4)
+	if deviations.StaticRouteAddRemoveUnsupported(dut) {
+		t.Skip("StaticRouteAddRemove is unsupported")
```

Comment:
A test case cannot be skipped . A alternate functionality via cli or alternate path has to be added here. 
Please refer to 'https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/README.md'